terraform/services/github-actions/deploy-roles/main.tf

```
@@ -0,0 +1,51 @@
+data "aws_iam_policy" "developer_boundary_policy" {
+  name = "developer-boundary-policy"
+}
+
+resource "aws_iam_policy" "runner" {
+  name = "github-action-runner"
+
+  description = "Runner has access to everything within the permissions boundary to allow for terraform and other deploy steps"
+
+  policy = <<EOT
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "*",
+      "Resource": "*"
+    }
+  ]
+}
+EOT
+}
+
+resource "aws_iam_role" "github-actions-runner-role" {
+  name                 = "github-actions-runner-role"
+  permissions_boundary = data.aws_iam_policy.developer_boundary_policy.arn
```

Set path on the role:

```suggestion
  name = "github-actions-runner-role"
  path = "/delegatedadmin/developer/"

  permissions_boundary = data.aws_iam_policy.developer_boundary_policy.arn
```